package.json

```diff
@@ -69,5 +72,12 @@
     "plugins": [
       "transform-class-properties"
     ]
+  },
+  "lint-staged": {
+    "src/**/*.js": [
+      "prettier --print-width=120 --single-quote --trailing-comma=all --write",
```

Comment:
let's stick in a huge int then. It's 2017, if you are the kind of person who has mastered vim or emacs, you chose to be in control. I see no reason to accomodate limitations in those editors that removed with a few trivial plugin installs.